.scalafmt.conf

```diff
@@ -5,6 +5,7 @@ style = defaultWithAlign
 docstrings                 = JavaDoc
 indentOperator             = spray
 maxColumn                  = 120
+lineEndings                = preserve
```

Comment:
Please don't change the formatting rules

Reply:
But I am using Windows, when compile, many file changes.
With this just `keep` the same.

Reply:
Many files is currnetly end with `CRLF`